tests/__fixtures__/cardanoSignTransaction.js

```diff
@@ -91,6 +91,21 @@ const SAMPLE_OUTPUTS = {
         address: 'addr_test1vr9s8py7y68e3x66sscs0wkhlg5ssfrfs65084jrlrqcfqqtmut0e',
         amount: '1',
     },
+    token_output: {
+        address: 'addr1q84sh2j72ux0l03fxndjnhctdg7hcppsaejafsa84vh7lwgmcs5wgus8qt4atk45lvt4xfxpjtwfhdmvchdf2m3u3hlsd5tq5r',
+        amount: '2000000',
+        tokenBundle: [
+            {
+                policyId: "95a292ffee938be03e9bae5657982a74e9014eb4960108c9e23a5b39",
+                tokenAmounts: [
+                    {
+                        rawAssetName: "74652474436f696e",
+                        amount: "7878754"
+                    }
+                ]
+            }
+        ]
+    }
```

Comment:
Maybe it does not matter, but I find it safer to have a test with at least two tokens and two assets in a group.